src/osp/Active/SysNewton.cpp

```diff
@@ -626,10 +619,33 @@ void SysNewton::on_shape_destruct(ActiveReg_t& reg, ActiveEnt ent)
 
 void SysNewton::on_world_destruct(ActiveReg_t& reg, ActiveEnt ent)
 {
+
+
     NewtonWorld const *world = reg.get<ACompNwtWorld>(ent).m_nwtWorld;
     if (world != nullptr)
     {
-        NewtonDestroyAllBodies(world);
```

Comment:
When NewtonDestroyAllBodies was previously called, it would invalidate all the pointers in the ACompCollider, leading to a crash when they're destructed.

Reply:
Ugh. That's a terrible interface. Shame on newton.